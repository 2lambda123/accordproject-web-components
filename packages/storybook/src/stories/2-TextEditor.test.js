import initStoryshots from '@storybook/addon-storyshots';

initStoryshots({
  storyKindRegex: /Text Editor/,
  suite: 'Text Editor Storyshots'
});
