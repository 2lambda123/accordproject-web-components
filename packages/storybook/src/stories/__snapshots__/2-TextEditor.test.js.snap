// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text Editor Storyshots Text Editor Demo 1`] = `
<div>
  <div
    className="ui cards"
  >
    <div
      className="ui fluid card"
      id="ap-text-editor"
      onClick={[Function]}
    >
      <div
        className="content"
      >
        <textarea
          className="textarea"
          onChange={[Function]}
          placeholder="This is the contents of the editor. Respond to the onChange notification to save updates."
          readOnly={false}
          style={
            Object {
              "height": 0,
            }
          }
          value="This is the contents of the editor. Respond to the onChange notification to save updates."
          width="100%"
        />
      </div>
    </div>
  </div>
</div>
`;
